app/page.js

```diff
@@ -1,29 +1,57 @@
+'use client';
 import Image from 'next/image';
 import styles from './page.module.css';
+import { useState, useEffect } from 'react';
+
+export default function Home() {
+  const [fact, setFact] = useState(null);
+
+  useEffect(() => {
+    // Fetch an initial fact when the component mounts
+    ApiCall().then((initialFact) => {
+      setFact(initialFact);
+    });
+  }, []);
+
+  const fetchNewFact = () => {
+    // Fetch a new fact and update the state
+    ApiCall().then((newFact) => {
+      setFact(newFact);
+    });
+  };
 
-function Bs(props) {
   return (
-    <div className={`${styles.card} ${styles.center}`}>
-      <ul>
-        <li>{props.element} -Sushi Roll</li>
-      </ul>
-    </div>
+    <main className={styles.main}>
+      <QuoteBox fact={fact} />
+      <Button onClick={fetchNewFact} />
+    </main>
   );
 }
 
-export default async function Home() {
-  const res = await fetch('https://catfact.ninja/fact');
-  const str = await res.json();
-  const arry = ['Skywalker Roll', 'Rock & Roll', 'Puyallup Roll'];
+async function ApiCall() {
```

Comment:
Typically, when you're writing functions in javascipt they should start with a lowercase letter because we prefer [camel case](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#camel-case). 

Also, this function syntax is okay, but I'm kinda a picky person and would prefer it if you could create it as an anonymous function. https://www.geeksforgeeks.org/javascript-anonymous-functions/

```js
const getFact = async () => {
   // ...Do Things
}
```
Something of that nature.


To add onto this to help you not confuse yourself, when you're creating components in React, it is normal to capitalize the function name!